utilities/storage.py

```
@@ -631,6 +633,35 @@ def write_file(vm, filename, content, stop_vm=True):
+        wait_timeout=TIMEOUT_5MIN,
+        sleep=TIMEOUT_5SEC,
+        func=run_ssh_commands,
+        exceptions_dict={SSHException: []},
+        host=vm.ssh_exec,
+        commands=shlex.split(command),
+        tcp_timeout=tcp_timeout,
+    )
+    try:
+        for sample in samples:
+            if sample:
+                LOGGER.info(f"SSH command executed successfully: {sample}")
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)